apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -1,17 +1,29 @@
 package no.nav.etterlatte.avkorting
 
 import com.fasterxml.jackson.databind.JsonNode
+import no.nav.etterlatte.beregning.Beregning
 import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
 import no.nav.etterlatte.libs.common.periode.Periode
 import no.nav.etterlatte.libs.common.tidspunkt.Tidspunkt
+import java.time.YearMonth
 import java.util.*
 
 data class Avkorting(
     val avkortingGrunnlag: List<AvkortingGrunnlag>,
     val avkortingsperioder: List<Avkortingsperiode>,
-    val avkortetYtelse: List<AvkortetYtelse>
+    val avkortetYtelse: List<AvkortetYtelse>,
+    val aarsoppgjoer: List<Aarsoppgjoer>
 ) {
-    fun leggTilEllerOppdaterGrunnlag(nyttGrunnlag: AvkortingGrunnlag): Avkorting {
+
+    fun kopierAvkorting(): Avkorting = nyAvkorting(
+        avkortingGrunnlag = avkortingGrunnlag.map { it.copy(id = UUID.randomUUID()) }
```

Comment:
Får man samme problem på avkorting som på det @henrikg  nevnte på utviklersync for vilkårsvurdering++?

Reply:
Avkorting blir mer likt beregning og beregningsgrunnlag. Vi kopierer grunnlagene fra forrige behandling og beregner beregning og avkorting på nytt.